### ISSUE TITLE
Exclude user from hashtag feed

### ISSUE THREAD

**Issue by kerim** [mastodon/mastodon#20936]
### Pitch

When following a hashtag there are often users whose posts are only marginally relevant to the hashtag and yet use it all the time. One option would be to block or mute the user entirely, but I don't want to do that - I just want to exclude them from the hashtag when following or viewing pinned on the advanced view. 

### Motivation

Right now it seems one can only exclude certain keywords not people?

**Comment by keithjgrant** [comment]
Absolutely. I would use the heck out of this feature if it were available.

I don't always want to mute the user entirely — if they post something interesting and somebody boosts it into my timeline, I'd still like to see it, but I'd like to be able to trim the noise.

**Comment by FlominatorTM** [comment]
That's basically the opposite of https://github.com/mastodon/mastodon/issues/34977 ... nice
